src/js/components/Settings/VoterPhoneVerificationEntry.jsx

```diff
@@ -302,10 +302,12 @@ class VoterPhoneVerificationEntry extends Component {
     // console.log('cancelShouldCloseModal:', cancelShouldCloseModal);
     if (cancelShouldCloseModal) {
       this.closeSignInModalLocal();
-    } else if (isCordova() || isMobileScreenSize()) {
-      // if (this.props.showAllSignInOptions) {
-      //   this.props.showAllSignInOptions();
-      // }
+    } else if (isMobileScreenSize()) {
+      if (this.props.showAllSignInOptions) {
+        this.props.showAllSignInOptions();
+      } 
+    } else if (isCordova()) {
```

Comment:
Hi @omgitsmiles unfortunately "else if" isCordova has to be first so that the Cordova devices that are mobile don't get picked up by the first "else if (isMobileScreenSize)".

Reply:
Understood, will fix.